dnsdist/conf/dnsdist.conf

```diff
@@ -33,3 +33,11 @@ carbonServer(os.getenv('DESEC_NS_CARBONSERVER'), os.getenv('DESEC_NS_CARBONOURNA
 
 -- DoT
 addTLSLocal('0.0.0.0', '/etc/dnsdist/dot.cer', '/etc/dnsdist/dot.key')
+
+-- cache
+pc = newPacketCache(100000, {
+    maxTTL=86400, minTTL=0,
+    temporaryFailureTTL=60, staleTTL=60,
+    dontAge=false,
```

Comment:
[Should be true](https://dnsdist.org/reference/config.html#packetcache).

Reply:
Good catch!